roles/base/defaults/main.yml

```diff
@@ -12,8 +12,9 @@ base_users:
     #dotfiles: false
 base_user_groups: ['sudo']
 
-base_dotfiles_url: "https://github.com/Flowm/dotfiles.git"
-base_dotfiles_dir: "/home/{{ base_user.name }}/.myconf"
+base_dotfiles_url: "{{ base_user.dotfiles_url }}"
+base_dotfiles_dir: "/home/{{ base_user.name }}/{{ base_user.dotfiles_dir }}"
+base_dotfiles_install_cmd: "/home/{{ base_user.name }}/{{ base_user.dotfiles_dir }}/{{ base_user.dotfiles_install_cmd }}"
```

Comment:
Did you try this with multiple users?

I think these 3 params would need to be contained within the base_users list above in order to support different options for every user.

Reply:
tried with two users and debug message:
```
- name: Debug
  ansible.builtin.debug:
    msg:
    - "User: {{ base_user }}, Dotfiles dir is {{ base_dotfiles_dir }}"
```
seems to work:

> Dotfiles dir is /home/flow/.myconf

> Dotfiles dir is /home/rupert/dotfiles

Reply:
if e.g. dotfiles_dir is not defined for the second user, ansible fails with 
> 'dict object' has no attribute 'dotfiles_dir'
this would be ok, right?